### PR TITLE
View and like/unlike articles

### PR DESCRIPTION
**Pull request by joedixon**
Full page screen shot

### CODE REVIEW

File: resources/css/articles.css [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+    @apply my-8;
+}
+
+.article img {
+    @apply shadow-lg;
+}
+
+.article p code {
+    @apply bg-green-light px-1 text-base;
+}
```
Comment (driesvints) [inline resources/css/articles.css:34]: I think we still need quite a few styling extra's here. Think blockquotes, h4-6, links.

It's best that you try to search for Markdown examples and try to render them. Like this one: http://www.unexpected-vortices.com/sw/rippledoc/quick-markdown-example.html

File: resources/views/livewire/like-article.blade.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<div>
+    @if (Auth::guest())
```
Comment (driesvints) [inline resources/views/livewire/like-article.blade.php:2]:
```suggestion
    @guest
```

File: resources/views/livewire/like-article.blade.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        <div class="text-gray-600 mr-2 py-2 inline-block flex items-center">
+            <span class="text-3xl mr-3">👏</span>
+            {{ $this->article->likesCount() }}
+        </div>
+    @else
+        <button type="button" wire:click="toggleLike" class="text-green-dark mr-2 py-2 flex items-center">
+            <span class="text-3xl mr-3">👏</span>
+            {{ $this->article->likesCount() }}
+        </button>
+    @endif
```
Comment (driesvints) [inline resources/views/livewire/like-article.blade.php:12]:
```suggestion
    @endGuest
```

**Comment by joedixon** [comment]
All_the_Markdown_Laravel_mobile

All_the_Markdown_Laravel

**Pull request merged**